wsdl-parser-cli/src/main.rs

```diff
@@ -38,18 +40,18 @@ fn main() {
         process_dir(input_path, output_path)
     } else {
         process_single_file(input_path, output_path)
-    }
-    .map_err(|e| println!("Error: {}", e))
-    .unwrap();
+    }?;
```

Comment:
Can you please put two `?`s after the two function calls rather than one after the if-else expression? This looks rather unusual.